<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UR3e Simulator </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="URInputPort1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Port 30002 Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="URInputPort2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Port 30003 Server</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Welcome to UR3e Simulator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/res/icon/resources/ur3e-image.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>UR3e TCP</string>
    </property>
    <widget class="QLineEdit" name="x_tcp">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>30</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.390575474</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="y_tcp">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-0.003665399</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="z_tcp">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.138074747</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rx_tcp">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1.461644235</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ry_tcp">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.707841340</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rz_tcp">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>-0.626511390</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X (m)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y (m)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z (m)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RX (rad)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RY (rad)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RZ (rad)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendNewTCP">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send new TCP</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
